tests/property/test_hash.py

```diff
@@ -28,6 +30,27 @@ def test_basic_hash_properties(self, a: ElementModQ, b: ElementModQ):
         if ha != hb:
             self.assertNotEqual(a, b)
 
+    @given(elements_mod_p())
+    def test_hash_of_big_integer(self, input: ElementModQ) -> None:
+        """Test hashing of larger integers such as element mod p"""
+
+        # Arrange.
+        input_hash = hash_elems(input)
+        invalid_hex = "0" + input.to_hex()
+        leading_zeroes = "00" + input.to_hex()
+
+        # Act.
```

Comment:
This looks like two tests in one.  I don't know if this is idiomatic for Python, but personally, I prefer multiple tests with a single act clause each. That way when a test fails you can read the name of the test and know what went wrong.